src/components/tx/SendFromBlock/index.tsx

```diff
@@ -14,9 +14,9 @@ const SendFromBlock = (): ReactElement => {
         Sending from
       </Typography>
 
-      <Typography variant="body2">
+      <Box sx={({ typography }) => typography.body2}>
```

Comment:
Alternatively, it could be `<Typography variant="body2" component="div">`.

What about the Typography right above?

Reply:
I've specified the `component` instead.

The `Typography` above contains text.